deploy/playbooks/roles/common/tasks/ssl.yml

```diff
@@ -0,0 +1,99 @@
+---
+
+- name: ensure ssl certs directory
+  action: file dest=/etc/ssl/certs state=directory
+  sudo: true
+
+- name: ensure ssl private directory
+  action: file dest=/etc/ssl/private state=directory
+  sudo: true
+
+- name: check for SSL cert
+  stat: path=/etc/ssl/certs/{{ fqdn }}-bundled.crt
+  ignore_errors: true
+  register: ssl_cert
+  sudo: true
+
+- name: check for SSL key
+  stat: path=/etc/ssl/private/{{ fqdn }}.key
+  ignore_errors: true
+  register: ssl_key
+  sudo: true
+
+- name: copy SSL cert
+  copy:
+    src: "{{ ssl_cert_path }}"
+    dest: "/etc/ssl/certs/{{ fqdn }}-bundled.crt"
+    mode: 0777
+  when:
+    - ssl_cert.stat.exists == false
+    - development_server == true
+  sudo: true
+  notify: restart nginx
+
+- name: copy SSL key
+  copy:
+    src: "{{ ssl_key_path }}"
+    dest: "/etc/ssl/private/{{ fqdn }}.key"
+  when:
+    - ssl_key.stat.exists == false
+    - development_server == true
+  sudo: true
+  notify: restart nginx
+
+- name: ensure letsencrypt acme-challenge path
+  file:
+    path: "{{ ssl_webroot_path }}"
+    state: "directory"
+    mode: 0755
+  when:
+    - development_server == false
```

Comment:
We could avoid all the ``when`` clauses with ``development_server == false`` if we just moved them into a ``ssl_prod.yml`` and used that ``when`` clause on the ``include``.